libraries/Microsoft.Bot.Builder/Teams/TeamsInfo.cs

```diff
@@ -44,6 +44,22 @@ public static async Task<TeamsMeetingParticipant> GetMeetingParticipantAsync(ITu
             }
         }
 
+        /// <summary>
+        /// Gets the details for the given meeting id.
+        /// </summary>
+        /// <param name="turnContext"> Turn context.</param>
+        /// <param name="meetingId"> The BASE64-encoded id of the Teams meeting.</param>
+        /// <param name="cancellationToken"> Cancellation token.</param>
+        /// <returns>Team Details.</returns>
+        public static async Task<MeetingInfo> GetMeetingDetailsAsync(ITurnContext turnContext, string meetingId = null, CancellationToken cancellationToken = default)
```

Comment:
Note: I don't *love* that the method name is `...Details`, but the return type is `...Info`, but the object returned is:

```json
{
	"details": <theDetailedInformation>,
	"conversation": <conversationAccountForTheMeeting>,
	"organizer": <meetingOrganizer'sChannelAccount>
}
```

...so it contains a "details" property key that contains more specific meeting details and it makes sense to me that *that* would take the "Details" name.

Open to discussion.

Reply:
Per Teams, the method will change to `getMeetingInfo`, so I'll make a couple method name changes.